cypress/integration/plugins/customImportMap/0_before.spec.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+
+import { testIndexDataSet, BASE_PATH } from '../../../utils/constants';
+
+describe('Before', () => {
+  before(() => {
+    cy.deleteAllIndices();
+
+    localStorage.setItem('home:welcome:show', 'true');
+
+    cy.visit(`${BASE_PATH}/app/home#/tutorial_directory/sampleData`);
```

Comment:
looks like you try to add sample data at line 16 - 26, you could call `miscUtils.addSampleData()` directly, it will handle adding all 3 sample datasets and check if it's successful ([source code](https://github.com/opensearch-project/opensearch-dashboards-test-library/blob/6ab6553db640b70f72bb1fabe5c1802636842be9/common-utils/misc-utils.js#L51-L60))
also if you add sample data, you need to remove it after testing, see [example](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/integration/common/dashboard_sample_data_spec.js#L100-L106), otherwise other tests will fail because they can't add sample data

Reply:
Thanks, updating it!